### ISSUE TITLE
delegation power may be off

### ISSUE THREAD

**Issue by fisx** [liqd/aula#685]
when playing with the delegation browser, i noticed that the power of a student on global scope is increased when that student has power in some class scope (or i think i did, needs reproducing).

this would be wrong: the voting power is the number of votes that a user can cast in a certain delegation scope, would there be a voting event in that scope.  the votes of sub-scopes must not be taken into account on the super-scope, because they only count for *some*, but not *all* sub-scopes.

in other words: the power of a student is the power in a given scope, plus the power in all parent scopes (where not overridden by child-scopes).  the child scopes of the given scope do not affect it.

**Comment by fisx** [comment]
hypothesis: voting power is calculated correctly, but the edges in the delegation network browser are pointing from target to source (delegate to delegatee).  needs confirmation.

**Comment by fisx** [comment]
confirmed: edges are from delegate to delegatee.  should be easy enough to fix.
